values.yaml

```diff
@@ -11,6 +11,9 @@ fullnameOverride: ""
 ## Application configuration.
 config: {}
 
+## Labels applied to all manifest objects for the Chart.
+globalLabels: {}
```

Comment:
Just for my own clarity - this would only ever stamp the top level of objects, and not traverse down into the specs and update those. :+1: 

I traced it, but wanted to be explicit in what I think I understand :)

Reply:
It appears that way to me

Reply:
yup, only top level objects. that felt right to me, but this can be updated to traverse down if we want that too

Reply:
nah, That would have concerned me as K8s api between revisions has different rules about duplicated labels in different contexts. That can be problematic in its own right, as kyler has found with annotations, which really have a different ruleset. (should be treated like a set, but are more like a weird set)